Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
Maybe a good enough check is seeing if `BATCH_LENGTH` equals `POINTS_LENGTH`, or something similar.